gpii/configs/shared/gpii.config.couch.development.startWithCleanData.json5

```diff
@@ -0,0 +1,33 @@
+/*
+
+    A configuration that starts a Couch instance if needed on startup, and which always clean up the data if the
+    database already exists.  Note, you can control how Couch is started/stopped using an environment variable:
+
+    * (nothing, the default): Start a Docker container running CouchDB.
+    * `GPII_TEST_COUCH_USE_VAGRANT`: If this is set, create a Vagrant VM that runs Docker and CouchDB.
+    * `GPII_TEST_COUCH_USE_EXTERNAL`: If this is set, connect to an external instance of CouchDB running on port 25984.
+
+    For more information, see the documentation for the gpii-couchdb-test-harness package:
+
+    https://github.com/GPII/gpii-couchdb-test-harness
+
+*/
+{
+    "type": "gpii.config.couch.development.startWithCleanData",
```

Comment:
I just noticed [gpii/configs/shared/gpii.config.couch.development.manualTesting.json5](https://github.com/GPII/universal/blob/master/gpii/configs/shared/gpii.config.couch.development.manualTesting.json5), which is quite similar with this config.

Reply:
Hi, @cindyli.  As we discussed on IRC, the manual testing grade starts from a different "harness", one that is designed to persist its data, that always tries to start couch if needed, but that never tries to provision it.  This config is distinct in that it both attempts to start couch and always cleans the data on startup.  A key difference is removing the cleanDbs: false option from the manual testing config.

Reply:
I see. Thanks.